test/playground.html

```diff
@@ -2,7 +2,10 @@
 <html lang="en">
 	<title>O hai</title>
 
-	<div class="foo" id="foo">foo</div>
+	<button aria-description="Undefined global">myVar</button>
+
+	<div id="aria-describedby">Undefined global</div>
+	<button aria-describedby="aria-describedby">myVar</button>
```

Comment:
Did you mean to make this change?

Reply:
No... I always forget to revert the playground changes.